README.rst

```diff
@@ -27,10 +27,15 @@ is a swell project, if I may say so myself.
 "I hate reading, make this quick."
 ==================================
 
-Before doing anything, install a recent version of `Docker`_ to your machine. Next,
-clone a clusterdock topology to your machine. For this example,
-we'll use the `nodebase topology`_.
-Assuming that you've already installed **clusterdock**, you could start a 2-node cluster:
+Before doing anything, install a recent version of `Docker`_ to your machine and install
+Clusterdock as:
+
+.. code-block:: console
+
+    $ pip install clusterdock
```

Comment:
We may want to use `pip3` as I realized that some Python newbies don't realize they were installing it into Python 2 and having issues.